Style.css

```diff
@@ -604,7 +612,8 @@ a {
 
 @media screen and (min-width: 992px) {
   .navbar {
-    padding: 0 210px;
+    margin-top: 1em;
+    margin-left: 50px;
```

Comment:
- [ ] Kindly remove these margins to the navbar because is making your project to overflow horizontally. if you want to put some spaces around it kindly use the padding because this will not mess up your code.


![image](https://user-images.githubusercontent.com/65068771/160034991-78cb4dc2-d23f-4854-8f81-a96ac173a205.png)